stories/topics/proc-gcp-set-monitoring-logging-yml.adoc

```diff
@@ -11,11 +11,11 @@ If you do not require a parameter, remove that parameter from the configuration
 * `cloud_credentials_path` is the absolute path toward your credentials. 
 This must be an absolute path.
 * `deployment_name` is the name of the deployment.
-* `components` is an array containing the type of component where you want to do the setup. 
+* `components` (Optional) is an array containing the type of component where you want to do the setup. 
```

Comment:
Optional: components (in bacticks) is an array containing...

'the type of component where you want to do the setup' doesn't make much sense. 'the type of component on which you want to carry out the setup' ?

Reply:
I used this format for the optional fields to remain consistent with the rest of the docs. I feel either we make this change everywhere or we keep it like this